src/test/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotTest.java

```diff
@@ -945,6 +946,75 @@ public void verifyConnectorFailsIfMarkSnapshotDoneFails(boolean consistentSnapsh
         YugabyteDBSnapshotChangeEventSource.FAIL_WHEN_MARKING_SNAPSHOT_DONE = false;
     }
 
+    @Test
```

Comment:
Don't we need to parameterise this test to run with older stream as well?

Reply:
@asrinivasanyb suggested to run it consistent snapshot stream only.